### DISCUSSION TITLE
https://saicharankandukuri.github.io/posts/gnome-in-android/

### DISCUSSION THREAD

**Discussion by giscus** [SaicharanKandukuri/SaicharanKandukuri.github.io#5] [Blog]
# https://saicharankandukuri.github.io/posts/gnome-in-android/

Running Gnome 42 in termux proot inside ubuntu 22.04 jammy is possible now

https://saicharankandukuri.github.io/posts/gnome-in-android/

**Comment by SaicharanKandukuri** [comment]
**Update**
In new package update gnome wont work anymore in `proot`

But luckily, i made a backup of working gnome and made it available in `udroid` ( ubuntu-on-android )
```bash
udroid install jammy:gnome
```

**Comment by wangerzi** [comment]
The `termux` and `udroid` projects are so cool, I almost played all night when I first learned about them a few days ago, and the next day was Monday o(╥﹏╥)o
